com.roguetemple.HyperRogue.json

```diff
@@ -0,0 +1,57 @@
+{
+  "app-id":"com.roguetemple.HyperRogue",
+  "runtime":"org.freedesktop.Platform",
+  "runtime-version":"20.08",
+  "sdk":"org.freedesktop.Sdk",
```

Comment:
21.08?

Reply:
yeah it's out now. It wasn't when the PR was opened.